script/utils/diamondEMERGENCYPauseGitHub.sh

```
@@ -0,0 +1,154 @@
+
+  # go through all networks and start background tasks for each network (to execute in parallel)
+  RETURN=0
+  for NETWORK in "${NETWORKS[@]}"; do
+      handleNetwork "$NETWORK" "$PRIVATE_KEY_PAUSER_WALLET" &
+  done
+
+  # Wait for all background jobs to finish
+  wait
+  # Check exit status of each background job
+  for JOB in $(jobs -p); do
+    wait $JOB || RETURN=1
```

Shellcheck

[warning] RETURN appears unused. Verify use (or export if used externally).

(SC2034)